app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_user_comment_counter
+  def update_user_comment_counter
+    user.update(comment_counter: post.comments.count)
```

Comment:
- [ ] The `post` is what has a comments counter, not `user`.

- [ ] Kindly run a migration to add `comments_counter` to the posts table. This method will raise errors if you call it as there is no such column on your table.

  ![image](https://github.com/Surafels/Blog-app/assets/54848363/d28e6345-45eb-4603-a0e3-c178e50366d8)